src/garage/torch/algos/trpo.py

```diff
@@ -99,8 +98,8 @@ def _compute_objective(self, advantages, valids, obs, actions, rewards):
 
         return surrogate
 
-    def _optimize(self, itr, obs, actions, rewards, valids, baselines):
+    def _optimize(self, obs, actions, rewards, advantages):
         self._optimizer.step(
-            f_loss=lambda: self._compute_loss(itr, obs, actions, rewards,
-                                              valids, baselines),
-            f_constraint=lambda: self._compute_kl_constraint(obs))
+            f_loss=lambda: self._compute_loss(obs, actions, rewards, advantages
```

Comment:
please override this formatting so that it's less strange

Reply:
fixed